### ISSUE TITLE
[FEATURE] Adding Dark and light theme

### ISSUE THREAD

**Issue by Sibam-Paul** [yashchaudhari008/minime#41]
![image](https://github.com/user-attachments/assets/83a034de-d814-47af-bf93-ab6a590adf19)
There is no icon  to change the theme of the page 

#### Expected:

<img width="710" alt="image" src="https://github.com/user-attachments/assets/a74d91dc-7ea1-4ebd-95fe-ccdcad5e4a8d">
<img width="722" alt="image" src="https://github.com/user-attachments/assets/9e583bec-4ede-4b34-8b33-76d8dd6fe08b">

**Comment by yashchaudhari008** [comment]
Agree! I was thinking of adding it as an option to the settings page, which is currently in progress. [#28](https://github.com/yashchaudhari008/minime/issues/28)

**Comment by yashchaudhari008** [comment]
Hey @Sibam-Paul, which browser are you using? and is it incognito mode? because it seems fonts are not loaded correctly

**Comment by Sibam-Paul** [comment]
@yashchaudhari008 brave

**Comment by Sauhard74** [comment]
is the issue open ? if yes I want to contribute can you assign me this

**Comment by yashchaudhari008** [comment]
Yes, it's open but it can only be picked after #28
